### PR TITLE
Adding Styling

### PR DESCRIPTION
**Pull request by SnakyMz**
Things I have done:

- [ ] Added missing JSX elements
- [ ] Added mobile version of styling
- [ ] Added desktop version of styling

### CODE REVIEW

File: src/components/AddBook.jsx
Comment (Efiamotu-1) [inline src/components/AddBook.jsx:1]: - [ ] I get this error shown in the image below when I try to run your project, please kindly fix this error and try resubmitting so i can see that other styles for each book card has been implemented 👍 
rer

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @SnakyMz , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.jsx
Comment (SnakyMz) [reply src/components/AddBook.jsx:1]: please provide the image

Comment (SnakyMz) [reply src/components/AddBook.jsx:1]: Can we huddle please? the website is working perfectly on my terminal

Comment (Efiamotu-1) [reply src/components/AddBook.jsx:1]: @SnakyMz i can see that now, please resubmit

Comment (SnakyMz) [reply src/components/AddBook.jsx:1]: ok Thanks

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   return (
     <header>
-      <h1>Bookstore</h1>
       <nav>
+        <h1>Bookstore</h1>
         <ul>
-          <li><NavLink to="/" className="navs">Books</NavLink></li>
-          <li><NavLink to="/categories" className="navs">Categories</NavLink></li>
+          <li><NavLink to="/" className="navs">BOOKS</NavLink></li>
+          <li><NavLink to="/categories" className="navs">CATEGORIES</NavLink></li>
```
Comment (Alusp) [inline src/components/Navbar.jsx]: - [ ] Good job in implementing the changes requested by the previous reviewer, However, after a successful run, I noticed that your styling missed some guidelines from the given design, Please match your design to the given requirement, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention on the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

Srudent Zep


</td>
      <td>

BigZep

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @SnakyMz, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️
- [ ] UI Closer to the given design :heavy_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.jsx (hunk 1 above)
Comment (SnakyMz) [reply src/components/Navbar.jsx]: Hello @Alusp , thanks for the review
For the change in the bottom is it the category dropdown or the "ADD BOOK" button
Please reply
thanks

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @SnakyMz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes IMplemented :heavy_check_mark: 
- Very Nice Design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**